### ISSUE TITLE
Generate PDF of child information

### ISSUE THREAD

**Issue by rockfordlhotka** [HTBox/MobileKidsIdApp#234]
**AS A PARENT** I need the ability to generate a PDF containing all the information about my missing child. I can then send this PDF to law enforcement or other people I'm working with so that they have all the information necessary to assist in the process of finding and returning my child.

**TECHNICAL INFO** The PDF could be generated on the client device, or in an Azure service, but ultimately needs to be available for the end user to add to an email as an attachment.

**Comment by bseebacher** [comment]
Anyone know of any good cross-platform libraries to generate reports on a Xamarin app? Otherwise, we're going to have to push this to the cloud for rendering, which probably opens us up to some PII/regulation compliance headaches.

**Comment by rockfordlhotka** [comment]
Agreed, though I think the PII issues are minimal if we don't persist anything in the cloud, so it is data in flight (SSL) and in memory, and the result is a file on the phone.

**Comment by bseebacher** [comment]
Agreed - it's minimal risk, but something we should list in the terms / conditions / etc. (#62 #235)  Something to the effect of "server side processes may be used as part of the service - data will not be persisted", etc.

_(Here's the part where I mention that I am not a lawyer...)_

**Comment by binaryjanitor** [comment]
Grabbing this for ThatConference GiveCamp

**Comment by zhangxd6** [comment]
Did you look at the Essential Studio for Xamarin.Forms PDF Library by sync fusion? There is community license to allow to use it free.[license link ](https://www.syncfusion.com/products/communitylicense)Disclaimer: I did not personally use this one before.

**Comment by binaryjanitor** [comment]
Jon grabbing this
